src/dialog/savedqueries/dlgsavedquerieseditor.cpp

```diff
@@ -0,0 +1,50 @@
+#include <QSqlTableModel>
+
+#include "dialog/savedqueries/dlgsavedquerieseditor.h"
+#include "dialog/savedqueries/savedqueriestablemodel.h"
+#include "library/libraryfeature.h"
+#include "library/trackcollection.h"
+
+DlgSavedQueriesEditor::DlgSavedQueriesEditor(LibraryFeature* pFeature,
+                                             TrackCollection* pTrackCollection,
+                                             QWidget* parent)
+        : QDialog(parent),
+          m_pTrackCollection(pTrackCollection),
+          m_pFeature(pFeature) {
+    setupUi(this);
+    SavedQueriesTableModel *pSaveModel = 
```

Comment:
I can't deal with looking at all these naked new statements anymore, it's too hard to know when there's a memory leak and when there isn't as ownership is unclear at the point of declaration. I think I'll make that helper function you suggested the other day.